pallets/fungible/src/lib.rs

```diff
@@ -441,6 +523,7 @@ impl<T: Config> Pallet<T> {
 		Ok(allowance)
 	}
 
+	/// Transfers of tokens that `from` gave to `spender` to manage, to `to` ID.
```

Comment:
This might seem a bit convoluted to people, could you rephrase it like "Transfers parts of a token as a non-owner account, with approval of the owner of the token (see {link to set_allowance()}.\n\n# Arguments\n\n- `collection`: ...". The same for `burn_from`. You can borrow the particular descriptions of the arguments of these two from somebody else who's done the token pallets.

Reply:
Or rather, use the refungible description, it's pretty good without the need to describe the arguments.